File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/CheckWaitlistStatus.java
package me.grishka.houseclub.api.methods;

import me.grishka.houseclub.api.ClubhouseAPIRequest;

public class CheckWaitlistStatus extends ClubhouseAPIRequest<CheckWaitlistStatus.Response> {
    public CheckWaitlistStatus() {
        super("POST", "check_waitlist_status", Response.class);
    }

    public static class Response {
        public boolean isWaitlisted, isOnboarding;
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/CheckWaitlistStatus.kt
package me.grishka.houseclub.api.methods

import me.grishka.houseclub.api.ClubhouseAPIRequest

class CheckWaitlistStatus :
    ClubhouseAPIRequest<CheckWaitlistStatus.Response?>("POST", "check_waitlist_status", Response::class.java) {
    class Response(var isWaitlisted:Boolean = false, var isOnboarding:Boolean = false)
}